css/themes/peter_river.css

```diff
@@ -481,7 +481,7 @@ button.btn.btn-default.navbar-btn.mv-navbar-btn:focus{
 
 #help .modal-content{
 	font-family: 'roboto_regular', Arial, Sans-serif!important;
-	height: 880px;
+	height: 770px;
```

Comment:
Do not change this here or push a request to the main mview repository because it's a themes from geobretagne